index.html

```diff
@@ -120,5 +120,40 @@ <h5>Next level iceland blue bottle fingerstache:</h5>
         </ul>
       </section>
     </section>
+    <section class="contact">
+      <h2>contact</h2>
+      <form>
+        <label for="name">name</label>
+        <input type="text" id="name">
+        <label for="email">e-mail</label>
+        <input type="email" id="email">
+        <label for="topic">topic</label>
+        <input type="text" id="topic">
+      </form>
+      <img src="assets/office.jpg" alt="office building image">
+      <p>
+        Our office is located<br/>
+        in the heart of Warsaw
+      </p>
+      <p>Feel free to say hello!</p>
+      <p>
```

Comment:
For addresses it is helpful to give the browser a bit of the information with metadata [eg schema](https://schema.org/address) because it for example allows Google to grab the data and show nice information about the company in their search results. Take a look how it is done on Lunar website in inspector. The result in Google is like that because of it:

![screen shot 2019-01-03 at 14 40 46](https://user-images.githubusercontent.com/1983291/50640598-9b1d1a00-0f65-11e9-9a82-6f835ee24db1.png)